public/components/getting_started/components/utils.tsx

```diff
@@ -21,18 +20,20 @@ const fetchAssets = async (tutorialId: string, assetFilter?: 'dashboards' | 'ind
   return responeData;
 };
 
-export const UploadAssets = async (tutorialId: string) => {
+export const UploadAssets = async (tutorialId: string, mdsId: string, mdsLabel: string) => {
   const { setToast } = useToast();
-  try {
-    const responeData = await fetchAssets(tutorialId);
+  const http = coreRefs.http;
 
-    const blob = new Blob([responeData.data], { type: 'application/x-ndjson' });
-    const file = new File([blob], 'ndjson-file.ndjson');
+  try {
+    const response = await http!.post(`/api/observability/gettingStarted/createAssets`, {
+      body: JSON.stringify({
+        mdsId,
+        mdsLabel,
+        tutorialId,
+      }),
+    });
 
-    const error = await uploadBundle(file);
-    if (error) {
-      console.error(error.message);
-    } else {
+    if (response) {
```

Comment:
Have we tested this? Does it create successful toast when the node API returns 500 or error?

Reply:
May be can check response.statusCode or something similar

Reply:
Tested and confirmed response on Aug 14